charts/super-agent-deployment/values.yaml

```diff
@@ -0,0 +1,110 @@
+# -- Override the name of the chart
+nameOverride: ""
+# -- Override the full name of the release
+fullnameOverride: ""
+
+# -- Name of the Kubernetes cluster monitored. Can be configured also with `global.cluster`.
+cluster: ""
+# -- This set this license key to use. Can be configured also with `global.licenseKey`
+licenseKey: ""
+# -- In case you don't want to have the license key in you values, this allows you to point to a user created secret to get the key from there. Can be configured also with `global.customSecretName`
+customSecretName: ""
+# -- In case you don't want to have the license key in you values, this allows you to point to which secret key is the license key located. Can be configured also with `global.customSecretLicenseKey`
+customSecretLicenseKey: ""
```

Comment:
Not sure if this is going to be implemented here (perhpas inside sub-agents configs), I think i would remove it to avoid noise or missleading config for now

Reply:
Actually the SuperAgent needs a license key, therefore it makes sense to have this, however I am not sure if it can be passed already as a envVar

Reply:
Thinking about this again, are we adding the logic to leverage this?

If the super-agent supports it we need something like this https://github.com/newrelic/nri-kubernetes/blob/0c8d886317af6bcfeac7536565e654378da0b652/charts/newrelic-infrastructure/templates/kubelet/daemonset.yaml#L116

Reply:
and the same for licenseKey and customSecretName